pages/osx/mkfile.md

```diff
@@ -0,0 +1,15 @@
+# mkfile
+
+> Create one or more empty files of any size.
+
+- Create an empty file of 15 kilobytes:
+
+`mkfile -n {{15k}} {{file_name}}`
+
+- Create a file of a given size and unit (bytes, kB, MB, GB):
```

Comment:
Does the `k` of `kB` have to be lowercase? All the others are uppercase.

Reply:
👍